assets/scss/components/_navbar.scss

```diff
@@ -12,48 +13,66 @@
 
   @include m-up {
     display: flex;
-    height: 75px;
-    padding: 0 40px;
+    font-size: 14px;
+    height: 63px;
+    padding: 0 35px 0 48px;
   }
 
   &__header {
     align-items: center;
+    background-color: $white;
     display: flex;
     flex-wrap: wrap;
     justify-content: space-between;
     min-height: 75px;
```

Comment:
There should be height:75px, not min-height:75px (height of navbar can't be bigger)